src/Renderer/AtlasBuilder.js

```diff
@@ -0,0 +1,94 @@
+import * as THREE from 'three';
+import Capabilities from '../Core/System/Capabilities';
+import fit from './Packer';
+
+export default function pack(images, uvs) {
```

Comment:
My understanding is that you pack all the `images` into one big texture (with the help of the canvas thing below) and that uvs contains the individual coordinates. Is that right?